components/ui/button.tsx

```diff
@@ -9,15 +9,18 @@ const buttonVariants = cva(
   {
     variants: {
       variant: {
-        default: "bg-primary text-primary-foreground hover:bg-primary/90",
-        destructive:
-          "bg-destructive text-destructive-foreground hover:bg-destructive/90",
+        // Built-in variants
+        // default: "bg-primary text-primary-foreground hover:bg-primary/90",
         outline:
           "border border-input bg-background hover:bg-accent hover:text-accent-foreground",
+        destructive:
+        "bg-destructive text-destructive-foreground hover:bg-destructive/90",
         secondary:
-          "bg-secondary text-secondary-foreground hover:bg-secondary/80",
+        "bg-secondary text-secondary-foreground hover:bg-secondary/80",
         ghost: "hover:bg-accent hover:text-accent-foreground",
         link: "text-primary underline-offset-4 hover:underline",
+        // Custom variants
+        default: "bg-orange-600 text-white hover:bg-orange-600/90",
```

Comment:
Move this back to the first line item, as it's the default look and feel.

Reply:
Done. Committed and will be pushed when all other changes are complete.